ScoreSaber/ScoreSaber.csproj

```diff
@@ -4,7 +4,8 @@
     <ProjectGuid>{844132E4-CDFC-49D3-950D-38F62ED8B513}</ProjectGuid>
     <TargetFramework>net48</TargetFramework>
     <OutputType>Library</OutputType>
-    <LangVersion>8.0</LangVersion>
+    <LangVersion>9</LangVersion>
+    <Nullable>disable</Nullable>
```

Comment:
Why explicitly disable this if you are gonne reenable it in tons of files? We should be consistent here and either enable it everywhere or disable it everywhere.

Reply:
enabled it